### DISCUSSION TITLE
Effect gets activated after input refresh rate change

### DISCUSSION THREAD

**Discussion by robinkoehler** [awawa-dev/HyperHDR#170] [Q&A]
Hi Awawa and everyone! 

There is some occasional strange behaviour on my setup after switching refresh rates. I'm outputting to my TV at 60hz by default. I have PotPlayer and MadVR configured to switch refresh rate to match movie refresh rate for stutter free playback.

Sometimes when switching from 60hz to 23.976hz (Blu-ray 24p), or when switching back from 24p to 60hz, HyperHDR seems to automatically activate an effect. Here is a clip:

https://youtu.be/PjbuNgLt7oY

The issue is usually resolved by manually changing refresh rate in windows, and switching back immediately. Or by playing a movie again, which switches to 24p, and then exiting the movie.

I do not know if this is somewhat related to HyperHDR, or to my specific setup. I do have an HDMI splitter with 1080p downscaler in the mix, which sends 4k to the TV from the PC, and 1080p to the capture card attached to the raspberry pi for HyperHDR.

Thank you and best wishes,

Robin

**Comment by awawa-dev** [comment]
Hi
All the needed information are in logs. Probably an issue with a grabber that cause loss of the video stream, in some cases 'auto-resume' in the grabber could help.
Regards
Awawa

**Reply by robinkoehler** [reply]
Hi Awawa,

Ok thank you! I'll try that and will report back if I find anything in the logs.

Take care and best wishes,

Robin
